### ISSUE TITLE
Change default keys forever

### ISSUE THREAD

**Issue by renyhp** [fph/bastet#19]
Is there a way to customize keys and then retain this preference?

If not, can this be added? A simple implementation idea would just be to not regenerate `~/.bastetrc` if there is already one.

Thanks!

**Comment by fph** [comment]
Hello! What do you mean by 'forever'? Persist across several runs of the game? It should already work like that. If it doesn't on your platform, it must be a bug. Please add more information, so we can try to track it down.

**Comment by renyhp** [comment]
Yes, I mean persist across several runs of the application. This is the workflow:
* run `bastet`
* customize keys
* play a game (customized keys work)
* play another game (customized keys still work)
* exit via `Ctrl+C`
* run `bastet`
* play a game
* the previous setting has been forgotten!

I am running on Xubuntu 20.04.2, with bash and guake.

**Comment by fph** [comment]
Ah -- found the reason; apparently they only get saved if you quit using the menu, not with `Ctrl+C`. I'll see if I can fix it, thanks for the report.

**Comment by renyhp** [comment]
In fact, this probably raises the underlying problem that I don't really see a better way to quit the game while in game. Maybe have a key that quits just like the menu does (which also saves the keys)?
